src/Project/BuilderCfgFile.ts

```diff
@@ -207,8 +211,13 @@ export class BuilderCfgFile extends EventEmitter implements helpers.FileSelector
     return undefined;
   }
 
-  private onBeginImport() {
-    let cfgIni = helpers.loadCfgFile(this.cfgFilePath);
+  private onBeginImport(parsedCfg: any) {
+    // parsedCfg is came from 'Configuration-Settings'
+    // If parsedCfg is undefined, it means 'ONE: cfg import' was executed instead.
```

Comment:
That would be `parsedCfg: any|undefined` ?

Reply:
Yeah if 'ONE: cfg import' is executed, there won't be any value in `parsedCfg`.
But if `Run` is executed at `Configuration-Settings`, there will be value in `parsedCfg`